index.js

```diff
@@ -181,3 +185,53 @@ AddonManager.addInstallListener({
     app.send('addon-install:download-failed', formatInstallData(install));
   }
 });
+
+function pingServer(title, addon) {
+  cookieManager.getCookiesFromHost(prefs.HOSTNAME, function(cookies) {
+    let csrf;
+    let cookieStr = '';
+    cookies.forEach(function(c) {
+      cookieStr += c.name + '=' + c.value + ' ';
```

Comment:
Individual key-value pairs inside the HTTP Cookie header should be separated by a semicolon, so you could just do `cookies.join(':')` and not need to loop over the cookies array.